### DISCUSSION TITLE
execution_date not being rendered after upgrading to 2.2.1

### DISCUSSION THREAD

**Discussion by AnithaGG** [apache/airflow#19601] [Q&A]
Hi, we recently upgraded airflow from 2.1.0 to 2.2.0 and DAGs aren't running as expected. All these DAGs were added before the upgrade itself and they were running fine.
We use execution_date parameter in SparkSubmitOperator which was rendering fine before the upgrade fails now returning None

`"{{ (execution_date if execution_date.microsecond > 0 else dag.following_schedule(execution_date)).isoformat() }}"`

DAG run fails with the error

`jinja2.exceptions.UndefinedError: 'None' has no attribute 'isoformat'
`

Tried wiping out the database and ran as a fresh DAG but still same error 

Any help would be appreciated

**Comment by inception-cto** [comment]
I think I am facing the same issue. https://github.com/apache/airflow/discussions/19655
